app/views/posts/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Posts</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Content</th>
+      <th>Grass tags</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+
+<!-- update to hide posts that are 1) only_followers && 2) the current_user is not following.  Also update so edit and destroy are only availalbe to the user whose post it is -->
+
+    <% @posts.reverse.each do |post| %>
+      <tr>
+        <td><%= post.content %></td>
+        <td><%= post.grass_tags %></td>
+        <td><%= link_to 'Show', post %></td>
+        <td><%= link_to 'Edit', edit_post_path(post) %></td>
+        <td><%= link_to 'Destroy', post, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
+
+<br>
+
+<%= link_to 'New Post', new_post_path %>
```

Comment:
This file will very likely need a lot of edits, but I wanted to get this PR in and good to go because it had, like 500 commits in it.